x-pack/legacy/plugins/lens/public/editor_frame_plugin/editor_frame/index.scss

```diff
@@ -30,6 +30,20 @@
   position: relative;
 }
 
+.lnsSidebar__header {
+  padding: $euiSizeS 0;
+
+  > * {
+    display: flex;
+    align-items: center;
+  }
+}
+
+.lnsChartSwitch__summaryIcon {
+  margin-right: $euiSizeS;
+  transform: translateY(-2px);
```

Comment:
Assume this is to pixel push an alignment?

Reply:
Yeah, we need to work on our icon to text alignments